packages/schemas/src/foundations/jsonb-types/file-upload.ts

```diff
@@ -0,0 +1,12 @@
+import { number, object, string, type z } from 'zod';
+
+export const uploadFileGuard = object({
+  filepath: string(),
+  mimetype: string(),
+  originalFilename: string(),
```

Comment:
if we have the full path for the file in the zip, what's the purpose of storing filename here?

Reply:
removed